README.md

```diff
@@ -123,7 +123,7 @@ Output:
 
 This package is compatible with the following MeiliSearch versions:
 
-- `v0.10.X`
+- `v0.11.X`
```

Comment:
Weird that this change is in this branch because it is in the main branch `bump-meilisearch-v0.11` -> https://github.com/meilisearch/meilisearch-js/pull/413/files

Can you rebase:
```bash
$ git checkout bump-meilisearch-v0.11
$ git pull origin bump-meilisearch-v0.11
$ git checkout faceting
$ git rebase bump-meilisearch-v0.11
```

Reply:
Well I m pointing this branch to `bump...` and I also made a branch based on `bump..` so it is that weird I think

Reply:
I don't know what you did but it disappeared!
Thanks! 😬